GameData/RealismOverhaul/Engine_Configs/Agena_XLR81_Config.cfg

```diff
@@ -374,20 +376,20 @@
 			PROPELLANT
 			{
 				name		= UDMH
-				ratio		= 0.4492
+				ratio		= 0.209513
 				DrawGauge	= False
 			}
 
 			PROPELLANT
 			{
 				name		= IRFNA-III
-				ratio		= 0.5508
+				ratio		= 0.790487
```

Comment:
Looks like the math isn't working out right? Stated MR is 2.94:1 in the change, but actual MR as you've supplied here is (0.790487 x 0.001658) / (0.209513 x 0.000791) = 7.908. My guess is the other MR changes will also be busted, then.
To get an MR of 2.94:1 you do (ox_ratio x ox_density) / ((1-ox_ratio) x fuel_density) = MR and solve for ox_ratio. Wolfram says the equation is ox_ratio = fuel_density x MR / (fuel_density x MR + ox_density), and fuel_ratio = 1-ox_ratio, was that what you used?

(The result should be 0.5834 for IRFNA-III and 0.4166 for UDMH for MR 2.94:1)

Reply:
The calculation I had used to find the config ratio values was from an (albeit old) screenshot of Kirk saying its:

> ratio_volume = (MR * fuel_density / ox_density)
> ox_ratio = (ratio_vol / (ratio_vol + 1))
> then subtract from 1 to find fuel ratio

I'll double check them with the equation you provided

Reply:
The fact the ratio doesn't add up to 1 indicates you did the math wrong somewhere.

Reply:
Checked the other values I calculated against the `ox_ratio = fuel_density x MR / (fuel_density x MR + ox_density)` equation NK gave, and:

LR87-AJ-3 is correct, 2.02:1 at 0.5921 : 0.4078
LR87-AJ-9 is correct, 1.925:1 at 0.544 : 0.456
LR87-AJ-11A is correct, 1.91:1 at 0.542 : 0.458
Agena 8096 was incorrect, changed to 2.8:1 at 0.5718 : 0.4282
Agena 8096-39 was incorrect, changed to 2.94:1 at 0.5382 : 0.4618
Viking IV was incorrect, changed to 1.7:1 at 0.4812 : 0.5188
A-6 was correct, 1.219:1 at 0.4734 : 0.5266
Changed the A-7A ratio to match A-6, since they share the same fuel now
A-7B (before the A-7) was incorrect, changed to 1.354:1 at 0.5051 : 0.4949

I think I had switched the oxygen and fuel densities around by accident on the Agena engines...